scalableminds/webknossos
# Add volume undo/redo support

This PR adds support for Undo / Redo of volume annotations, while still supporting skeleton undo in e.g. hybrid tracings.

### URL of deployed dev instance (used for testing):
- https://addvolumeundo.webknossos.xyz

### Steps to test:
- Open a hybrid tracing
- randomly draw around and create some node/trees
- Use undo/redo as you wish. Undo should redo your last skeleton action or your last brush stroke.
- Then save the tracing and reload it. Everything should look the same after reloading.


### Issues:
- fixes #4001

------
